src/test/java/cs361/battleships/models/GameTest.java

```diff
@@ -15,7 +15,12 @@ public void testPlaceTurns() {
         game.placeShip(3, new Square(0, 1), false);
         assert(game.opponentsBoard.ships.size() == 2);
     }
-
+    @Test
+    public void testSimpleAttack() {
```

Comment:
Thank you for adding this test. Game.java should now pass the line requirement for tests.